lib/oracledb.js

```diff
@@ -85,6 +85,9 @@ function _initCLib() {
     './node_modules/oracledb/' + nodbUtil.RELEASE_DIR + '/' + nodbUtil.BINARY_FILE,
     './node_modules/oracledb/' + nodbUtil.RELEASE_DIR + '/' + 'oracledb.node'
   ];
+  if (arg1 !== undefined && arg1.binaryPath !== undefined) {
```

Comment:
Here you can simply check `if (options.binaryPath !== undefined)` if you pass `options` through to this function.

Reply:
@anthony-tuininga Thanks for your advice. Changes has been updated for using options directly.

Reply:
There's no point making changes unless you sign the [OCA](https://github.com/oracle/node-oracledb/blob/main/CONTRIBUTING.md) !  (Sorry!)

Reply:
Sorry I just noticed that. Already submitted and under review. Do I need to close this PR until review completed?

Reply:
It fine to leave this open - we have the labels to indicate the overall OCA status

Reply:
Looks like it was approved so now we just need the technical part reviewed.

As a procedural note, we would copy this PR to our private repo, and then sync the changes in that private repo back here.  I.e this PR would be closed without any apparent merge but the changes will eventually land.